README.md

```diff
@@ -18,6 +21,27 @@ We write about everything that happens to the PrestaShop software:
 
 This devblog also hosts How-Tos on certain features, before they find their way in the Developer Guide or the Designer Guide.
 
+## Run locally
+
+build.prestashop.com is hosted by GitHub Pages and generated by [Jekyll](https://jekyllrb.com/) which is a Ruby static site generator.
+
+We recommend using Ruby 2.5 or 2.6.
```

Comment:
```suggestion
We recommend using Ruby 2.5.8
```
as it's defined here https://github.com/github/pages-gem/blob/master/.ruby-version